tests/extension/messages/cellOutput.test.ts

```diff
@@ -78,4 +80,4 @@ suite('Handle CellOutput messages', () => {
     test('Vercel cell output', async () => {
         return expectOutput(OutputType.vercel)
     })
-})
```

Comment:
what is the standard for us having an empty line at the end?

Reply:
#595